src/components/WwwFrame/TheFooter.vue

```diff
@@ -3,36 +3,58 @@
 		<nav class="small-footer hide-for-large" aria-label="Footer navigation">
 			<ul class="hide-for-print">
 				<li>
-					<router-link :to="applyUrl">
+					<router-link
+						:to="applyUrl"
+						v-kv-track-event="['Footer', 'click-Borrow-mobile view']"
```

Comment:
My instinct is that we could have the same tracking events for mobile and desktop

Reply:
Maybe double check with @brandonatkiva

Reply:
You're instincts were right @ryan-ludwig. I connected with Brandon and he requested that the mobile links be mapped to analogous options from the desktop version of the footer. Changes made in latest commit.